packages/portal-proto/src/features/cases/CasesView/CasesCohortButton.tsx

```diff
@@ -125,7 +125,7 @@ export const CasesCohortButton: React.FC<CasesCohortButtonProps> = ({
           onClose={() => setShowCreateCohort(false)}
           onActionClick={(newName: string) => {
             setName(newName);
-            if (numCases > 1) {
+            if (numCases > 0) {
```

Comment:
I'm pretty sure this conditional was in place because we want the filters to contain the case id when there is only 1 case instead of the set id so that the case id is displayed in the filter representation. I will try to dig up the ticket to verify but that would mean this is not the correct change to fix the bug (i.e we actually do not want to create a set when only one case is selected).

Reply:
Good memory!  If there is only one case in the cohort, we wanted to show the case id in the query expressions rather than "X input cases". That said, if that's a pain to do, we can probably live with "1 input case" for now (well, except it'll be inconsistent when we are filtering via the custom filters...:( )